packages/elements-core/src/components/TryIt/state.ts

```diff
@@ -0,0 +1,5 @@
+import { IServer } from '@stoplight/types';
```

Comment:
I don't know if like naming this file `state`.

It kind of suggests that this is the whole state that TryIt has, meanwhile I believe it already uses more atoms (?)

I actually like the idea of having a dedicated file where you could glance at all the state that component has (and can share with other components). But this would be a larger work and as it is right now, I believe this file name simply misinforms the reader.

I would just stick to some `chosenServer` or `chosenServerAtom` name 🤷

Reply:
I don't mind either way - could collocate it with it's most relevant component (`ServerDropdown`) and ditch this file altogether? Or just re-name the file to `chosenServer` as you originally suggested - which do you prefer?

Reply:
Renamed to `chosenServer`